src/components/backend-ai-summary-view.ts

```diff
@@ -389,6 +390,8 @@ export default class BackendAISummary extends BackendAIPage {
           this.webui_version = globalThis.packageVersion;
           this.appDownloadUrl =
             globalThis.backendaiclient._config.appDownloadUrl;
+          this.fasttrackEndpoint =
+            globalThis.backendaiclient._config.fasttrack_endpoint;
```

Comment:
@agatha197 Please check `fasttrack_endpoint` to `CamelCase` format like others.

Reply:
reflected!